Array/ArrayWords.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+
+public class ArrayWords {
+    public static void main(String[] args){
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Insert at least three words, separated by comas and space");
+        String input = sc.nextLine();
+        String [] words = input.split(", ");
+        while (true) {
+            System.out.println("Chose the filter for your words:");
+            System.out.println("1. Show the words with maximum symbols length");
+            System.out.println("2. Show the words, which starts with the defined letter");
+            System.out.println("3. Show the words, which ends with the defined letter");
+            System.out.println("4. Show the words, which contains the defined letter");
+            System.out.println("0. Exit the filter");
+            String filter = sc.next();
+            System.out.println("Filter = " + filter);
+            if (filter.equals("0")){
+                System.out.println("Goodbye");
+                break;
+            }
+            if (filter.equals("1")) {
```

Comment:
Please be more attentive, in the task you need to ask for the length of the word and show all words that are shorter or equal to entered length